tests/integration/targets/acme_account/aliases

```diff
@@ -1,2 +1,11 @@
 shippable/cloud/group1
 cloud/acme
+
+# shippable/posix/group1
+# Skip all VMs, since we cannot talk to the ACME simulator from these:
+# (TODO: remove when ansible-core 2.12 is the earliest version we support)
```

Comment:
The reason is a bug which got fixed during the work for split-controller testing: https://github.com/ansible/ansible/issues/75711

The problem is that fixing it in earlier versions probably requires a larger amount of work, which is why it might never get done. I'll add a reference to that issue into the aliases files.

Reply:
See ba52c8146cf955a7fc9ad2ebac7ec892ae551358.

Reply:
Ah sorry, this is not the 100% correct explanation. That issue causes crashing when we add the target and the skips, but the skips are needed because there's another feature / bugfix (I guess it's debatable what it actually is) that ended up in ansible-test 2.12 that's not existing in earlier versions. It's mostly independent from split-controller testing, so just mentioning split-controller testing here isn't correct.